tests/cli/test_init.py

```diff
@@ -16,29 +14,209 @@
 from tests.cli.utils import assert_no_logging_messages_or_tracebacks
 
 
-@pytest.mark.xfail(
-    reason="This command is not yet implemented for the modern API",
-    run=True,
-    strict=True,
+@pytest.mark.parametrize(
+    "invocation,input",
+    [
+        ("--v3-api init", "Y\n"),
+        ("--v3-api --assume-yes init", ""),
+    ],
 )
 @mock.patch("webbrowser.open", return_value=True, side_effect=None)
```

Comment:
Nitpick: Since init is so much simpler now we can remove the mock webbrowser and all it's assertions for maintainability in this an other init tests.

Reply:
I was leaving this in as a check on regression - but I think it's pretty safe to remove them and will probably speed up the tests so 👍